### PR TITLE
Browser support

### PR DESCRIPTION
**Pull request by vercell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,20 +101,6 @@ It is also possible to load the bundled css files from a CDN:
 If a specific version is needed, insert the version number into the URL, ie.:
 `<link rel="stylesheet" type="text/css" href="https://unpkg.com/@youseedk/dna@1.0.0/ys-bundle.min.css" />` will load version 1.0.0.
 
-## Browser Support
```
Comment (kimblim) [inline README.md]: I don't think we should delete this section completely, but maybe we could just add a link that points to the browser-support page on dna.yousee.dk?

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+
+
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+
+## Partial support
+- Internet Explorer 11 for Windows <sup>[[4]](#ref4)</sup>
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: I don't think this is acceptable from a business standpoint. YouSee still have a lot of active IE11 users (8%, I think) and I think that requires full support.

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+
+
+## No support
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: Why have "No support" at all? We've just stated a complete "white list", so there is no need for a "black list" — especially one that contains Opera, which has the same rendering engine as Chrome, so our sites will work just fine in Opera. We just choose not to test in it, as the user base is so small, but that doesn't mean it's "not supported".

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 4]
```diff
@@ -1,12 +1,32 @@
...
+- Internet Explorer 10 and older
+- Opera and Opera Mini
+- Console and Television based browsers
+
+
+## Notes
+Latest version cover only the major release version, patches and minor updates is not taken into consideration.
+
+1. <a name="ref1"></a>Chrome, Firefox and Edge has a short regular release cycle, support for 2 prior version is necessarily.
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: This is invalid HTML — instead write the ordered list completely, like this:
```
<ol>
  <li id="ref1">....</li>
  <li id="ref2">....</li>
  etc
</ol>
```

Comment (kimblim) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: Also, the last part of the sentence should be:
"support for 2 prior versions is necessary."

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 5]
```diff
@@ -1,12 +1,32 @@
...
+2. <a name="ref2"></a>Safari has a longer release cycle, therefor only one prior version is necessarily to be supported.
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: .."therefore support for only one prior version is necessary."

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 6]
```diff
@@ -1,12 +1,32 @@
...
+3. <a name="ref3"></a>Samsung Internet is the new wildcard in the browser realm, luckily based on Chromium, however possible never updated by the consumer. As the result, only the latest release can be supported.
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: .."possibly never updated by the consumer. As a result, only the latest release can be supported."

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 7]
```diff
@@ -1,12 +1,32 @@
...
+4. <a name="ref4"></a>Partial support, means that small visuel bugs are acceptable, given that you can read and interact with the content.
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: "visual"

File: README.md (hunk 1 above)
Comment (grandorf) [reply README.md]: Yeah. I agree

File: src/docs/03-Code Guidelines/05-Browser-Support.md (hunk 2 above)
Comment (grandorf) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: We have agreed to stop full support for Internet Explorer 11. It still needs to work so we should not start to get incidents from customers. But if there is a few pixels off there and there we won't fix it. It has to look acceptable of cause but not perfect.

File: src/docs/03-Code Guidelines/05-Browser-Support.md (hunk 3 above)
Comment (grandorf) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: I like the idea with no support. But I think it should just be
```
Internet Explorer 10
Internet Explorer 9
Internet Explorer 8
```
Just to make it clear that we don't test in these browsers and won't fix bugs.

File: src/docs/03-Code Guidelines/05-Browser-Support.md (hunk 2 above)
Comment (grandorf) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: I think you should move `Internet Explorer 11` up to desktop browsers we support and then write (only partial support) and then at line 16 write what partial support is and even come up with an example maybe.

File: src/docs/03-Code Guidelines/05-Browser-Support.md (hunk 3 above)
Comment (vercell) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: So, keep the section or remove the section? For now I agree with Kim and it's now removed. However, lets discuss it further next time everybody is at the office.

File: src/docs/03-Code Guidelines/05-Browser-Support.md [hunk 8]
```diff
@@ -1,12 +1,23 @@
...
+## Phone & Tablet
+- Chrome (3 latest versions) for Android and iOS <sup>[[1]](#ref1)</sup>
+- Safari (2 latest versions) for iOS <sup>[[2]](#ref2)</sup>
+- Samsung Internet (latest version) for Android <sup>[[3]](#ref3)</sup>
+
+## Notes
+Latest version cover only the major release version, patches and minor updates is not taken into consideration.
+
+<ol>
+    <li id="ref1"></a>Chrome, Firefox and Edge has a short regular release cycle, support for 2 prior version is necessarily.</li>
```
Comment (kimblim) [inline src/docs/03-Code Guidelines/05-Browser-Support.md]: @vercell 
There's still a closing `</a>` element here ;)

Comment (vercell) [reply src/docs/03-Code Guidelines/05-Browser-Support.md]: Embarrassing! it's now fixed.

**Pull request merged**